Sources/MongoSwift/MongoClient.swift

```diff
@@ -208,16 +211,20 @@ public class MongoClient {
 
     /**
      * :nodoc:
-     * Create a new client from an existing `mongoc_client_t`.
+     * Create a new client from an existing `mongoc_client_t`. The new client will destroy the `mongoc_client_t` upon
+     * deinitialization.
      * Do not use this initializer unless you know what you are doing.
      *
      * If this client was derived from a pool, ensure that the error api version was set to 2 on the pool.
      *
      * - Parameters:
-     *   - fromPointer: the `mongoc_client_t` to store and use internally
+     *   - pointer: the `mongoc_client_t` to store and use internally
      */
-    public init(fromPointer: OpaquePointer) {
-        self._client = fromPointer
+    public init(stealing pointer: OpaquePointer) {
+        // Initialize mongoc. Repeated calls have no effect so this is safe to do every time.
```

Comment:
do we actually need to call this here? I don't see it being called anywhere in the mongoc_embedded code but it feels like you would call it *before* creating a client there...

Reply:
Yeah, it's definitely supposed to be called before a client is created, at least with the normal driver. I'm not sure what the effect is doing so afterwards.

Reply:
per my messages in #swift-private... I think we should just call `mongoc_init` directly in the embedded library since it needs to happen before this initializer is used. added a note about that to the docstring for this.

Reply:
opened [SWIFT-458](https://jira.mongodb.org/browse/SWIFT-458) about this.